hnn_core/network_builder.py

```diff
@@ -575,10 +642,47 @@ def _parnet_connect(self):
                         nc_dict['A_weight'] = drive_conn[
                             receptor]['A_weight']
                         self._connect_celltypes(
-                            drive['name'], target_cell_type,
+                            connectivity,
                             drive_conn['location'], receptor, nc_dict,
                             unique=drive['cell_specific'])
 
+    # Generate connectivity list with all to all connections
```

Comment:
```suggestion
```

it's a bit non-standard to add a comment above the function/method. I know it's elsewhere in the codebase but we should avoid it

Reply:
Oops, forgot to remove this after adding the docstring